src/Core/Timer/ITimer.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+
+namespace Chronos.Timer.Core
+{
+    public interface ITimer
+    {
+        public TimeSpan ElapsedTime { get; }
```

Comment:
Can you please add summaries to all properties / methods? This is especially important for the interfaces.